packages/@aws-cdk-testing/framework-integ/test/core/test/integ.enable-additional-metadata.js.snapshot/EnableTelemtryStack.template.json

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
I could not see the metadata resource in this template, although you enabled the `ENABLE_ADDITIONAL_METADATA_COLLECTION` FF

Reply:
This is expected as all of our integration test have `analyticsReporting: false` to avoid contaminate the actual DB. I enabled to flag to make sure that it does deploy on my local test machine by overriding `analyticsReporting` to `true`.